docs/rawdata/raw_data_upload.md

```diff
@@ -8,7 +8,7 @@ This section gives an overview of how to upload data to measurements from QBiC's
 The following is required in order to successfully execute the measurement data upload.
 
 - Access to the project of interest
-- A SFTP client software (e.g. [FileZilla](https://filezilla-project.org)
+- A SFTP **client** software (e.g. [FileZilla](https://filezilla-project.org/download.php?type=client)
```

Comment:
```suggestion
- A SFTP **client** software (e.g. [FileZilla](https://filezilla-project.org)
```
There is a download button on their website... I think this link should go to the tool main page not to the download php